engine/flink/util/src/main/scala/pl/touk/nussknacker/engine/flink/util/metrics/MetricUtils.scala

```diff
@@ -0,0 +1,110 @@
+package pl.touk.nussknacker.engine.flink.util.metrics
+
+import java.util.ServiceLoader
+
+import cats.data.NonEmptyList
+import org.apache.flink.api.common.functions.RuntimeContext
+import org.apache.flink.metrics.{Counter, Gauge, Histogram, Meter, MetricGroup}
+import pl.touk.nussknacker.engine.flink.api.RuntimeContextLifecycle
+
+//Configure this as Service to use new reporting for Flink
+//this is a bit weird way, but it's temporary and passing configuration would be cumbersome
+trait UseNewMetrics
```

Comment:
Nice trick for now :+1: I wonder if in a near feature we can change deployment to use new flink rest api handling complex arguments. Then we can pass some environment vars and use it as a low level properties similar to `System.getProperty`